docs/UserGuide.md

```diff
@@ -401,7 +401,7 @@ App data are saved in the hard disk automatically after any command that changes
 
 ### Editing the data file
 
-App data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+App data are saved automatically as a JSON file `[JAR file location]/data/inSUREance.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
tiny nit here, but the JAR file uses uppercase `InSUREance.jar` while this is lowercase. Should we standardise to one of them (I am happy to change the name for JAR file too if we need to)

Reply:
lets standardise to `InSUREance`

Reply:
fixed